{{cookiecutter.project_shortname}}/scripts/server

```diff
@@ -2,6 +2,6 @@
 {% include 'misc/header.py' %}
 set -e
 
-export FLASK_DEBUG=1
-
-invenio run
+script_path=$(dirname "$0")
+invenio run --cert "$script_path"/../docker/nginx/test.crt \
```

Comment:
I think `--cert` and `--key` were [added in Flask v1.0.0](http://flask.pocoo.org/docs/1.0/changelog/#version-1-0).